### PR TITLE
Fix/pagination

### PR DESCRIPTION
**Pull request by btalma**

### CODE REVIEW

**Review by albertm805 — APPROVED** [review]
LGTM

File: dotnet/Controllers/RoutesController.cs [hunk 1]
```diff
@@ -220,7 +220,7 @@ public async Task<IActionResult> SetupHooks()
         public async Task<IActionResult> ListShipments()
         {
             Response.Headers.Add("Cache-Control", "private");
-            var response = await this._shipStationAPIService.ListShipments(string.Empty);
+            var response = await this._shipStationAPIService.ListShipments("shipDateStart=12-29-2021&shipDateEnd=12-29-2021&pageSize=500&page=4");
```
Comment (wender) [inline dotnet/Controllers/RoutesController.cs:223]: What is this hardcoded querystring?

File: dotnet/Services/VtexAPIService.cs [hunk 2]
```diff
@@ -233,12 +233,12 @@ public async Task<VtexOrder> GetOrderInformation(string orderId)
                 if (response.IsSuccessStatusCode)
                 {
                     vtexOrder = JsonConvert.DeserializeObject<VtexOrder>(responseContent);
-                    Console.WriteLine($"GetOrderInformation: [{response.StatusCode}] ");
+                    //Console.WriteLine($"GetOrderInformation: [{response.StatusCode}] ");
                 }
                 else
```
Comment (wender) [inline dotnet/Services/VtexAPIService.cs:238]: It seems that this ELSE isn't needed anymore

File: dotnet/Services/VtexAPIService.cs [hunk 3]
```diff
@@ -883,23 +884,29 @@ public async Task<string> ValidateShipments(string date, long page)
                                 listShipmentsTemp.Shipments.Add(shipment);
                                 bool success = await this.ProcessShipNotification(listShipmentsTemp);
                                 sb.AppendLine($"Order {orderId} {success}");
+                                if(success)
+                                {
+                                    Console.WriteLine($"Order {orderId} {success}");
+                                    _context.Vtex.Logger.Debug("ValidateShipments", null, $"Order {orderId}");
+                                }
                             }
                             else
```
Comment (wender) [inline dotnet/Services/VtexAPIService.cs:893]: Unnecessary ELSE condition

File: dotnet/Services/VtexAPIService.cs [hunk 4]
```diff
@@ -883,23 +884,29 @@ public async Task<string> ValidateShipments(string date, long page)
...
                             {
-                                sb.AppendLine($"invoice {shipment.ShipmentId} exists.");
+                                //sb.AppendLine($"invoice {shipment.ShipmentId} exists.");
                             }
                         }
                     }
                     else
```
Comment (wender) [inline dotnet/Services/VtexAPIService.cs:899]: Unnecessary ELSE condition

**Pull request merged**